README.rst

```diff
@@ -0,0 +1,57 @@
+cc2olx
+######
+
+*cc2olx* is a converter of `Common Cartridge <https://www.imsglobal.org/activity/common-cartridge>`_ `.imscc` files into `.tar.gz` files that can be imported into Studio.
+
+What is supported
+-----------------
+
+Converted:
+
+- HTML
+- Web links
+- Some videos
+- LTI links
+
+Not converted:
+
+- Images
+- QTI assessments
+
+
+Install
+---
+
+Clone repository and install via setup.py::
+
+    python setup.py install
+
+Use
+---
+
+The converter is a command-line Python 3 program.
+
+To convert one file::
+
+    cc2olx -i <IMSCC_FILE>
+
+This will write a `.tar.gz` file into the tmp directory here.
+
+To store all results in `zip` file::
+
+    cc2olx -r zip -i <IMSCC_FILE>
+
+To Do
+-----
+
+We would love help building out this tool.  If you can help, either with
+development, or with real-world testing, please get in touch: ned@edx.org.
```

Comment:
I think we might want to remove Ned's email here, but I will check in with the Open edX team.

Reply:
While Ned is a rock, no individual contributor should _ever_ be a single point of failure.

This was a note from when this was a hackathon project that we did _not_ expect would get official edx support.